frontend/device/input.lua

```diff
@@ -556,7 +561,7 @@ function Input:setTimeout(slot, ges, cb, origin, delay)
         -- If GestureDetector's clock source probing was inconclusive, do this on the UI timescale instead.
         if clock_id == -1 then
             deadline = time.now() + delay
-            clock_id = C.CLOCK_MONOTONIC
+            clock_id = C.CLOCK_BOOTTIME
```

Comment:
`CLOCK_BOOTTIME` is unfortunately not always available ;). (it's 2.6.39+ or something to that effect, maybe later on arm).

Reply:
That makes sense, yeah.

If we really want to be exhaustive, we can check for the availability of the clock source like we do for some of the clocks in time, but a simple heuristic like that should be fine ;).